.vscode/tasks.json

```diff
@@ -0,0 +1,37 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Build Documentation (Incremental)",
+            "type": "shell",
+            "command": "docker run -v \"./.git:/app/.git\" -v \"./docs:/app/docs\" -v \"./.artifacts:/app/.artifacts\" --platform linux/amd64 ghcr.io/elastic/docs-builder:edge",
+            "problemMatcher": [],
+            "group": {
+                "kind": "build",
+                "isDefault": true
+            }
+        },
+        {
+            "label": "Serve Documentation",
+            "type": "shell",
+            "command": "docker run -v \"./.git:/app/.git\" -v \"./docs:/app/docs\" -v \"./.artifacts:/app/.artifacts\" --platform linux/amd64  -p 8080:8080 ghcr.io/elastic/docs-builder:edge serve",
```

Comment:
Should this be `-p 3000:3000` now?

Reply:
Removed these for now while I investigate how to best expose these. 

Given the repository layout we need different mount strategy 

> docker run -v "./.git:/app/.git" -v ".:/app/docs" -v "./.artifacts:/app/.artifacts" -p 3000:3000 --expose 3000 ghcr.io/elastic/docs-builder:edge --force

Yields a few errors we need to address in `docs-builder` after we merge this.